servicetalk-buffer-internal/src/main/java/io/servicetalk/buffer/internal/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2018, 2021 Apple Inc. and the ServiceTalk project authors
```

Comment:
It looks like a new file, copyright year should be 2021 only

Reply:
gone